tests/dataframe/test_drop_pytest.py

```diff
@@ -15,17 +15,24 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
+import pytest
+
 # File called _pytest for PyCharm compatability
+from tests.common import TestData
 
 
-class TestDataFrameDrop:
+class TestDataFrameDrop(TestData):
     def test_drop(self, df):
-        df.drop(["Carrier", "DestCityName"], axis=1)
+        df.drop(labels=["Carrier", "DestCityName"], axis=1)
         df.drop(columns=["Carrier", "DestCityName"])
 
-        df.drop(["1", "2"])
-        df.drop(["1", "2"], axis=0)
+        df.drop(labels=["1", "2"])
+        df.drop(labels=["1", "2"], axis=0)
         df.drop(index=["1", "2"])
+        df.drop(labels="3", axis=0)
+        df.drop(columns="Carrier")
+        df.drop(columns=["Carrier", "Carrier_1"], errors="ignore")
+        df.drop(columns=["Carrier_1"], errors="ignore")
 
```

Comment:
Can we add an `assert` at the end of this test.

Reply:
So `conftest.py` takes care of it. It takes the df operation and applies it to both pandas and eland then asserts it. In this method. 
https://github.com/elastic/eland/blob/f241ae971aeb332c05c77fa3321b70175ea057e8/tests/conftest.py#L88

Am I missing something here ?